extensions/common/sql/sql-core/src/main/java/org/eclipse/dataspaceconnector/sql/store/AbstractSqlStore.java

```diff
@@ -0,0 +1,99 @@
+/*
+ *  Copyright (c) 2020, 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.sql.store;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.dataspaceconnector.spi.persistence.EdcPersistenceException;
+import org.eclipse.dataspaceconnector.spi.transaction.TransactionContext;
+import org.eclipse.dataspaceconnector.spi.transaction.datasource.DataSourceRegistry;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Objects;
+import javax.sql.DataSource;
+
+import static java.lang.String.format;
+
+public class AbstractSqlStore {
```

Comment:
```suggestion
public abstract class AbstractSqlStore {
```

Reply:
:+1: 
I've called it Abstract without abstract :facepalm:  :smile: